scripts.py

```diff
@@ -23,14 +24,15 @@
 from gutenberg import OCLCClassifyCoverageProvider
 from core.scripts import (
     Explain,
+    IdentifierInputScript,
     WorkProcessingScript,
     Script,
     RunMonitorScript,
 )
 from viaf import VIAFClient
 from core.util.permanent_work_id import WorkIDCalculator
 
-class RunIdentifierResolutionMonitor(RunMonitorScript):
+class RunIdentifierResolutionMonitor(RunMonitorScript, IdentifierInputScript):
     """Run the identifier resolution monitor.
```

Comment:
This script had its own custom way of parsing identifiers. I made it a real IdentifierInputScript.